docs/docker.md

```diff
@@ -0,0 +1,75 @@
+# Running a local docker instance
+
+**DISCLAIMER: this is still experimental and not recommended for production. Use at your own risk.**
+
+
+## Preperation
+
+Install docker for your operating system. If -God forbids- You are using windows, install docker machine as well and adjust all of the commands accordingly.
```

Comment:
the common idiom is "God forbid"... but maybe just leave the whole windows bit out?

Reply:
Leaving that part out would leave to some issues given that none of these commands work on windows directly. You either need a Linux subsystem or Cygwin or something similar or a docker-machine (which is a command) and then you need to modify all of the following commands on that guide. The other thing is just a joke, you can take it out if you want...

Reply:
sure, but it's not like you've included the windows commands anyway... you also need docker for mac if you're on a mac, etc.

Reply:
docker for mac doesn't have a different set of command, so installing docker would suffice.